extension/js/common/api/error/api-error.ts

```diff
@@ -0,0 +1,155 @@
+import { Xss } from '../../platform/xss.js';
+import { AjaxErr, ApiErrResponse, StandardError, StandardErrRes, AuthErr, GoogleAuthErr } from './api-error-types.js';
+import { Catch } from '../../platform/catch.js';
+
+export class ApiErr {
+    static eli5(e: any): string {
```

Comment:
Please use fat arrows everywhere :) `static eli5 = (e: any): string => {`

Then it should not complain about unbound methods when you do `.catch(ApiErr.reportIfSignificant)`

Reply:
Ohh, my bad
Will change